pkg/pillar/cmd/zedagent/zedagent.go

```diff
@@ -1972,6 +1993,7 @@ func handleZbootStatusImpl(ctxArg interface{}, key string,
 	}
 	log.Functionf("handleZbootStatusImpl: for %s", key)
 	// nothing to do
+	log.Errorf("in zboot impl, hasRealChange")
```

Comment:
Why error?

Reply:
Forgot to move this to Functionf. will change